Formula/esptool.rb

```diff
@@ -17,35 +17,33 @@ class Esptool < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "36ebd1053d6bd9e40b7d518970c5eb3da6fbb330e9530be81de02f3117cda416"
   end
 
+  # `pkg-config`, `rust`, and `openssl@1.1` are for cryptography.
   depends_on "rust" => :build
+  depends_on "cffi"
+  depends_on "openssl@1.1"
+  depends_on "pycparser"
   depends_on "python@3.11"
   depends_on "six"
 
-  resource "bitstring" do
-    url "https://files.pythonhosted.org/packages/d2/64/e733b18349be383a4b7859c865d6c9e5ccc5845e9b4258504055607ec1cb/bitstring-4.0.1.tar.gz"
-    sha256 "7719f08f6df89ce28453a5e580d4a8ec1d1bda892dbb033466da0ccd9bdcb706"
+  on_linux do
+    depends_on "pkg-config" => :build
```

Comment:
Skip the `on_linux` block; `pkg-config` will already be installed because of `rust`.

Reply:
👍

Reply:
as you can see, it does need `pkg-config`, https://github.com/Homebrew/homebrew-core/actions/runs/5108178647/jobs/9181831385?pr=132216

Reply:
Yes, I meant list it along with the other dependencies instead of separating it in an `on_linux` block.

Reply:
I see